spec/Spec/BlsEC/FQP.cry

```diff
@@ -289,9 +418,10 @@ property fqp_addition_commutative a b =
  * Verify that multiplication is commutative.
  * ```repl
  * :prove fqp_multiplication_commutative`{2}
+ * :prove fqp_multiplication_commutative`{12}
  * ```
  */
-fqp_multiplication_commutative : {n} (fin n, n >= 2) => FQP n -> FQP n -> Bit
+fqp_multiplication_commutative : {n} (fin n, n <= 12, n >= 2) => FQP n -> FQP n -> Bit
```

Comment:
This proof works with v4.13 of z3 but not 4.8. I am okay with this but we should document it.

Reply:
also... CVC5 works and appears to be faster for these.

You can add `:set prover=cvc5` in the docstrings for these.

Reply:
Done

Reply:
See last commit.